### ISSUE TITLE
Invalid generating INDEX name of tables in migrations

### ISSUE THREAD

**Issue by pankoffDaniel** [tortoise/aerich#195]
The problem happens when we change UNIQUE parameter in table field and launch migration.

Detail problem is below.

#models.py

```python
class Genre(Model):
    name = fields.CharField(max_length=255, unique=True)

    def __str__(self):
        return self.name
```

1. `aerich init-db`

PostgreSQL
![UNIQUE index is True](https://user-images.githubusercontent.com/26085462/132837380-2cd77769-09d7-4bf8-b12a-dd4c334c55be.png)

2. Change UNIQUE parameter `models.py`
3. `aerich migrate` generates this SQL:

```sql
-- upgrade --
DROP INDEX "idx_genre_name_c78cd9";
-- downgrade --
CREATE UNIQUE INDEX "uid_genre_name_c78cd9" ON "genre" ("name");
```

4. `aerich upgrade` raises an error:

```
tortoise.exceptions.OperationalError: index "idx_genre_name_c78cd9" does not exist
```

**Comment by madwayz** [comment]
Удали.

**Comment by waketzheng** [comment]
With "aerich>=0.8.0" index name is right. But if field was created by table schema, `aerich upgrade` will raise OperationalError -- tested by postgres:

```py
# cat migrations/models/1*.py
from tortoise import BaseDBAsyncClient


async def upgrade(db: BaseDBAsyncClient) -> str:
    return """
        DROP INDEX "uid_genre_name_c78cd9";"""


async def downgrade(db: BaseDBAsyncClient) -> str:
    return """
        CREATE UNIQUE INDEX "uid_genre_name_c78cd9" ON "genre" ("name");"""
```

**Comment by waketzheng** [comment]
Close as name error fixed, and create a new issue(#382) as `aerich upgrade` not work.
